ssl/ssl_lib.c

```diff
@@ -431,6 +431,35 @@ static int dane_tlsa_add(SSL_DANE *dane,
     return 1;
 }
 
+/*
+ * Return 0 if there is only one version configured and it was disabled
+ * at configure time.  Return 1 otherwise.
+ */
+static int ssl_check_allowed_versions(int min_version, int max_version)
+{
+    if (min_version == max_version
```

Comment:
I actually asked the same question. It would get a complex function, and I don't see an easy way to reuse the existing functions.

This is just a case of giving an error earlier, they'd get an error later anyway.